docs/LV_client/tools/mono_collision_plots.md

```diff
@@ -0,0 +1,29 @@
+# mono_collision_plots.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_collision_plots.png"/>
+</p>
+
+### Description
+
+Tool to plot the information provided by the collision sensor.
+
+For technical support contact us at <b>support@monodrive.io</b> 
+ 
+
+### Inputs
+
+- **Collision sensor sample:**  The accumulatative samples of the Collision sensor
```

Comment:
typo: accumulative

Reply:
Don't need the "The" and it feels really random when we decide to capitalize the sensor names and not- probably should choose one and stick with it: Accumulative samples of the Collision sensor.